x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -60,7 +60,7 @@ export function hasKibanaPrivilege(
 export const calculateEndpointAuthz = (
   licenseService: LicenseService,
   fleetAuthz: FleetAuthz,
-  userRoles: MaybeImmutable<string[]>,
+  userRoles: MaybeImmutable<string[]> = [],
```

Comment:
Set default values to avoid having errors if at some point these values are undefined because user is not authenticated.